### PR TITLE
June '19 update

### PR DESCRIPTION
**Pull request by broofa**
Fixes vulnerability warnings.
Keeps README from changing every time it's generated
Remove node v4 from travis (`mocha` no longer supports v4)

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+tag-version-prefix="v"
+save-prefix="~"
```
Comment (defunctzombie) [inline .npmrc]: would suggest exact versions rather than any floating version - but your call

Comment (broofa) [reply .npmrc]: 1. `package.json` already specifies the exact versions, and is what `npm install` prefers (I believe).
2. I actually see exact versions in package.json as an anti-pattern, as it makes it redundant with  `package-lock.json`, while nullifying any value to be had from the `npm outdated` and `npm upgrade` commands.

**Comment by broofa** [comment]
@defunctzombie Updated to address recent github audit warnings.  Approve, please.

File: package.json [hunk 2]
```diff
@@ -17,16 +17,16 @@
...
   },
   "devDependencies": {
-    "@commitlint/cli": "7.0.0",
-    "@commitlint/config-conventional": "7.0.1",
-    "eslint": "4.19.1",
-    "husky": "0.14.3",
-    "mocha": "5.2.0",
-    "runmd": "1.0.1",
-    "standard-version": "4.4.0"
+    "@commitlint/cli": "7",
```
Comment (defunctzombie) [inline package.json]: Personally not a fan of this style of dependency tracking (for same reasons as mentioned earlier in this PR).

**Pull request merged**